### PR TITLE
Happy Thoughts - Caroline Birgersson

### PR DESCRIPTION
**Pull request by carolinebirgersson**
Link to my deployed project: https://spread-happy-thoughts.netlify.app/

My reflections on how this week's project turned out: 
In the end I think it turned out good, but it was a struggle to merge Jennies and Vans thinking of how to approach this project with Maks thinking. I think I should have gone for Jennies thinking - and then give the site a new try with Maks approach instead of trying to combine it. It made everything a bit confusing and by Thursday evening I had it hard to understand my code.

Things I'd like to have clarified or explained in more detail:
How to add a url-file with a prop to the map "urls" in Maks approach.

### CODE REVIEW

**Review by lindahz — APPROVED** [review]
Caroline, you have done a great job with this project! 😍👏🏻 I love your happy thoughts styling. 

I think it was very brave of you not to settle and try a different approach, even if it felt confusing. I'm inspired by that way of thinking. ⭐️

Your code is clean and understandable, indentation looks great and it is semantically correct. I tested your website and it is working well on different devices. I also like the way you have structured your components and css in different folders. I always think it's a little scary with many folders (or to write the correct src), but I will dare to do that next time. 

I only have a few comments, honestly I could not find that much, great job!!! 😎

File: code/src/components/css/heartButton.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  cursor: pointer;
+  border: none;
+  width: 35px;
+  height: 35px;
+  border-radius: 50%;
+  padding: 0;
+}
+
+.heart-button:focus {
+  outline: transparent;
```
[the remark this answers is not in the document]
Comment (EmOlsz) [reply code/src/components/css/heartButton.css:22]: 👍

File: code/src/components/js/HeartButton.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+import "../css/heartButton.css";
+
+export const HeartButton = ({ onLiked, heart, heartId }) => {
+  const handleClick = () => {
+    fetch(
+      `https://happy-thoughts-technigo.herokuapp.com/thoughts/${heartId}/like`,
+      {
+        method: "POST",
+        body: "",
```
Comment (EmOlsz) [inline code/src/components/js/HeartButton.js:11]: No need for body over here, all essential data are being passed in the URL in line 8

File: code/src/components/js/ThoughtsInput.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    fetch(URL, {
+      method: "POST",
+      body: JSON.stringify({ message }),
+      headers: { "Content-Type": "application/json" },
+    })
+      .then(() => {
+        setMessage("");
+        onFormSubmit(message);
+      })
+      .catch((err) => console.log("error:", err));
```
Comment (EmOlsz) [inline code/src/components/js/ThoughtsInput.js:21]: `console.error()` would be more appropriate here

**Pull request closed without merging**